_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -34268,18 +34303,30 @@
 /obj/machinery/atmospherics/pipe/simple/supply/hidden{
 	dir = 4
 	},
-/turf/open/floor/plating,
-/area/maintenance/department/cargo)
+/obj/structure/disposalpipe/segment{
+	dir = 4
+	},
+/turf/open/floor/plasteel/purple/side,
+/area/science/circuit)
 "bFD" = (
 /obj/structure/cable{
 	icon_state = "4-8"
 	},
 /obj/machinery/atmospherics/pipe/simple/supply/hidden{
 	dir = 4
 	},
-/obj/effect/landmark/xeno_spawn,
+/obj/machinery/door/airlock/maintenance{
+	req_access_txt = "47";
+	req_one_access_txt = "0"
+	},
+/obj/structure/disposalpipe/segment{
+	dir = 4
+	},
+/obj/effect/mapping_helpers/airlock/cyclelink_helper{
+	dir = 8
+	},
 /turf/open/floor/plating,
-/area/maintenance/department/cargo)
+/area/science/circuit)
```

Comment:
/area/maintenance/department/cargo for the maint door's area.

Reply:
@ShizCalev Thanks for the reviews, I fixed the door's area.